server/lib/util.py

```diff
@@ -206,6 +206,15 @@ def get_cached_geojsons():
   return geojsons
 
 
+# Returns a homepage config in json format given the filename of the json file
+# for that config.
+def get_homepage_config_json(filename):
```

Comment:
just make this a general function: get_json(path_relative_to_root) ?

Reply:
done